lib/menu.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'price'
+class Menu
+  attr_accessor :data
+  def initialize(menu_data)
+   @data = menu_data
+  end
+
+  def print
```

Comment:
Did you consider abstracting this behaviour to another class?  Is it really the responsibility of the menu to print itself?

Reply:
I did think about having a general printing class - I think it would make sense to have this behaviour extracted. I just didn't get around to implementing it.